app/models/rules_of_origin/wizard_store.rb

```diff
@@ -0,0 +1,30 @@
+module RulesOfOrigin
+  class WizardStore < ::WizardSteps::Store
+    PREFIX = 'rules_of_origin-'.freeze
+    MAX_JOURNEYS = 5
+
+    class << self
+      def gc_store_data(backing_store)
+        backing_store.keys
+                     .select { |k| k.start_with? PREFIX }
+                     .sort_by { |k| backing_store[k]['modified'] || '2022-01-01T00:00:00Z' }
+                     .reverse
+                     .slice(MAX_JOURNEYS..)
+                     &.each { |k| backing_store.delete k }
```

Comment:
I suspect & is optional here?

Reply:
:eyes:

Reply:
Unfortunately slice returns nil, not an empty array if no matches

```
Loading development environment (Rails 7.0.4.3)
[1] pry(main)> [1,2,3,4].slice(5..)
=> nil
[2] pry(main)> 
```